pkg/appconsts/v3/app_consts.go

```diff
@@ -0,0 +1,7 @@
+package v3
+
+const (
+	Version           uint64 = 3
+	TxSizeCostPerByte uint64 = 10
+	GasPerBlobByte    uint32 = 8
+)
```

Comment:
should this also include all the other constants from v2?

```go
	SquareSizeUpperBound int    = 128
	SubtreeRootThreshold int    = 64
	// NetworkMinGasPrice is used by x/minfee to prevent transactions from being
	// included in a block if they specify a gas price lower than this.
	NetworkMinGasPrice float64 = 0.000001 // utia
```

Reply:
I'm not really sure. wdyt? @cmwaters

Reply:
Yes, that makes sense. 

I'm unsure why `NetworkMinGasPrice` needs to be a versioned constant. This seems just an initial constant that we set min fee to at genesis (or whenever the module begins to be used)

Reply:
yeah it's used as a default